Do not use redundant ^| table markers that OpenBSD groff rejects.

Index: doc/man/man5/UIL.5
--- doc/man/man5/UIL.5.orig
+++ doc/man/man5/UIL.5
@@ -1071,8 +1071,7 @@ allocation per use\&.
 .TS
 tab() box;
 c s s s
-l| l| l| l
-^| ^| ^| l.
+l| l| l| l.
 T{
 \fBData Storage Consumption for String Literals\fP
 T}
